tools/nvqpp/nvq++.in

```diff
@@ -448,6 +452,9 @@ while [ $# -ne 0 ]; do
 	--emulate | -emulate)
 		CUDAQ_EMULATE_REMOTE=true
 		;;
+	--disable-mlir-links | -disable-mlir-links)
+	    ENABLE_MLIR_LIB_LINKING=false 
+		;;
```

Comment:
Could you please add this option to the `show_help` function near line 150, too?

Reply:
I think I did, am I missing another spot? Line 158 there I put this 
```bash 
--disable-mlir-links
    Disable linking to cudaq-mlir-runtime and cudaq-builder. 
```

Reply:
Oops, you're right! Sorry about that! Too much task switching :)